source/crud/bulk-write.md

```diff
@@ -461,6 +461,13 @@ class BulkWriteException {
 }
 ```
 
+### Index Types
+
+The `insertResults`, `updateResults`, and `deleteResults` maps in `BulkWriteResult` and the `writeErrors` map in
+`BulkWriteException` specify `Index` as their key type. This value corresponds to the index of the operation in the
+`writeModels` list that was provided to `MongoClient.bulkWrite`. Drivers SHOULD use their language's standard numeric
+type for indexes for this type (e.g. `usize` in Rust). If no standard index type exists, drivers MUST use `Int64`.
```

Comment:
Why did this only come up in the Bulk Write spec? Would it make sense to amend the CRUD spec as well?

IIRC, Rust never implemented the old bulk write API. Did you not come across this with InsertManyResult?

Reply:
See this thread: https://mongodb.slack.com/archives/C035ZJL6CQN/p1724787735885729

It probably makes sense to update CRUD as well, although drivers probably won't make any changes to those types at this point. Rust already deviates from the spec and uses `usize` in `InsertManyResult`. Filed DRIVERS-2983.